dynamic_rest/serializers.py

```diff
@@ -66,19 +66,16 @@ def id_only(self):
     @property
     def data(self):
         """Get the data, after performing post-processing if necessary."""
-        if not hasattr(self, '_sideloaded_data'):
+        if not hasattr(self, '_processed_data'):
             data = super(DynamicListSerializer, self).data
-            if self.child.sideload:
-                self._sideloaded_data = ReturnDict(
-                    SideloadingProcessor(
-                        self,
-                        data
-                    ).data,
-                    serializer=self
-                )
-            else:
-                self._sideloaded_data = ReturnList(data, serializer=self)
-        return self._sideloaded_data
+            self._processed_data = ReturnDict(
```

Comment:
This would breaks a viewset that defines `sideload = False` and expects serialized data to be returned without an envelope. In effect, this is standard DRF behavior, so I'm not sure why you would be operating in this mode.